saleor/plugins/openid_connect/tests/test_utils.py

```diff
@@ -735,3 +736,114 @@ def test_assign_staff_to_default_group_and_update_permissions_update_user_permis
         permission_manage_users.name,
         permission_manage_orders.name,
     }
+
+
+@patch("saleor.plugins.openid_connect.utils.match_orders_with_new_user")
+@patch("saleor.plugins.openid_connect.utils.logger")
+def test__update_user_details_no_user_with_new_email_in_db(
```

Comment:
Nitpick but let's follow the convention of single underscores:
```suggestion
def test_update_user_details_no_user_with_new_email_in_db(
```